### PR TITLE
Remove JSON fetched STEP section; Add comment section

### PR DESCRIPTION
**Pull request by danielabreo**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,42 @@
...
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> data;
+
+  @Override
+  public void init() {
+    data = new ArrayList<String>();
+  }
+
+  private String convertToJson(ArrayList<String> messages) {
```
Comment (ccyang314) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:35]: nit: I think this function can convert to static.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,42 @@
...
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> data;
+
+  @Override
+  public void init() {
+    data = new ArrayList<String>();
```
Comment (ccyang314) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:32]: Is there any benefit constructing data varaible here vs. line 28?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -2,9 +2,17 @@ function changeImage() {
...
-  document.getElementById("greeting").innerText = greeting;
+function createListElement(text) {
+  const listElement = document.createElement('li');
+  listElement.innerText = text;
+  return listElement;
 }
-populateGreeting();
\ No newline at end of file
+
+function populateComments() {
```
Comment (ccyang314) [inline portfolio/src/main/webapp/script.js:11]: nit: add JSDoc comment to these functions.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -46,20 +46,27 @@
...
   <div class="container">
     <div class="row py-5 mb-3" style="height:100%">
       <div class="col-sm-12 my-auto mx-auto" style="padding-right:0px">
         <div class="container">
           <div class="row h-100 justify-content-center pt-5">
             <div class="col-md-6 my-auto">
-              <h2 id="greeting"></h2>
+              <h2>
+                Howdy! I'm Dan,
+              </h2>
```
Comment (brucezheng) [inline portfolio/src/main/webapp/index.html:69]: Optional/FYI: not every tag needs to break line. ```<h2>Howdy! I'm Dan,</h2>``` is fine if the tag contents are short.

**Comment by ccyang314** [comment]
Noticed the branch are keep `force-pushed`. Try amend and push instead of force push.

**Comment by danielabreo** [comment]
> Noticed the branch are keep `force-pushed`. Try amend and push instead of force push.

I tried this on a separate branch, and I think the force push is required if the commit being amended has already been pushed. Regular push works if the commit hasn't been pushed.

**Pull request merged**